dev/com.ibm.ws.jaxws.ejb/resources/com/ibm/ws/jaxws/ejb/internal/resources/JaxWsEJBMessages.nlsprops

```diff
@@ -1,27 +0,0 @@
-###############################################################################
-# Copyright (c) 2011 IBM Corporation and others.
-# All rights reserved. This program and the accompanying materials
-# are made available under the terms of the Eclipse Public License v1.0
-# which accompanies this distribution, and is available at
-# http://www.eclipse.org/legal/epl-v10.html
-#
-# Contributors:
-#     IBM Corporation - initial API and implementation
-###############################################################################
-
-#CMVCPATHNAME com.ibm.ws.jaxws.ejb/resources/com/ibm/ws/jaxws/ejb/internal/resources/JaxWsEJBMessages.nlsprops
-#ISMESSAGEFILE TRUE
-#COMPONENTPREFIX CWWKW
-#COMPONENTNAMEFOR CWWKW WebSphere JAXWS EJB Code
-#RANGE 0500 - 0599
-#
-#NLS_MESSAGEFORMAT_VAR
-#NLS_ENCODING=UNICODE
-#
-
-#{0} is the handler class name
-#{1} is the exception message
```

Comment:
These shouldn't have been deleted.  They should be in the 2.3 project.  This is why I suggested that since there is no code in the 2.3 project it is easier to just delete that one and update the bnd file for this project to be right.